rest/RESTClient.ts

```diff
@@ -14,16 +14,17 @@ export default class RESTClient {
   readonly client: Client;
   globallyRateLimited = false;
   https = new DiscordHTTPS(null, this);
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  queue: { method: HTTP_METHODS; endpoint: string; auth: boolean; payload?: { [s: string]: any } }[];
```

Comment:
```suggestion
  queue: { method: HTTP_METHODS; endpoint: string; auth: boolean; payload?: { [s: string]: any } }[] = [];
```
Also I think putting the type here as an interface would be easier to read